Chapter Zero PR (#191)
johko/computer-vision-course: chapters/en/Unit 0 - Welcome/welcome.mdx

# Welcome to the Computer Vision Course

> Dear learner,

Reviewer: ```suggestion
Dear Learner,
```

> Welcome to the Hugging Face course on computer vision. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!

Reviewer 2: ```suggestion
Welcome to the community-driven course on computer vision. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!
```

> Welcome to this course on computer vision. Whether it's unlocking your phone using facial identification, analyzing medical images to detect diseases, employing surveillance systems for public safety, monitoring wildlife, or generating new images, computer vision applications are adding tremendous value across a wide range of domains and Let's explore this world of computer vision together!

Reviewer: ```suggestion
Welcome to the Hugging Face course on computer vision. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!
```

Reviewer: Just Updated wording a bit to make it a bit clearer to reader.

> In this course, we will go from the very basics of computer vision all the way to some of the most recent advancements. The course is designed to touch on several foundation topics that cover the landscape of the field making computer vision what it is today.

Reviewer: ```suggestion
Throughout this course, we'll cover everything from the basics to the latest advancements in computer vision. It's structured to include various foundational topics, giving you a comprehensive understanding of what makes computer vision so impactful today.
```

Reviewer: Just Updated wording a bit to make it a bit clearer to reader.

> Before starting this course, we recommend you already have some python programming experience and you are familiar with transformers and machine learning (including neural networks). If you are not familiar with it, take a look on the [first unit of the NLP course](https://huggingface.co/learn/nlp-course/chapter1/3?fw=pt). Having a good grasp of pre-processing techniques and knowing mathematical operations (such as convolutions) is helpful, but it is not a requirement.

Reviewer: ```suggestion
Before beginning this course, it's recommended that you have some experience with Python programming and are familiar with transformers, machine learning, and neural networks. If these are new to you, consider reviewing the first unit of the Hugging Face NLP course](https://huggingface.co/learn/nlp-course/chapter1/3?fw=pt). While a strong knowledge of pre-processing techniques and mathematical operations like convolutions is beneficial, they are not prerequisites.
```

Reviewer: Just Updated wording a bit to make it a bit clearer to reader.

# Welcome to the Community Computer Vision Course

Dear learner,

Welcome to the community-driven course on computer vision. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!

Throughout this course, we'll cover everything from the basics to the latest advancements in computer vision. It's structured to include various foundational topics, making it friendly and accessible for everyone. We're delighted to have you join us for this exciting journey!

> ## How to start learning ✍️?

Reviewer 3: ```suggestion
In this page, you can find how to join the learners community, making a submission and getting a certificate, and more details about the course!
```

> To begin your learning journey, visit [huggingface.co/learn](http://huggingface.co/learn) and start exploring the chapters.

Reviewer 3: ```suggestion
```

I just noticed I made a mistake here, since we will have the course rendered there it's a bit confusing so we can remove this.

* **Unit 10 - Synthetic Data Creation** : discover the importance of synthetic data creation using deep generative models. Explore methods like point clouds and diffusion models and investigate major synthetic datasets and their applications in computer vision.
* **Unit 11 - Zero Shot Computer Vision** : delve into the realm of zero-shot learning in computer vision, covering aspects of generalization, transfer learning, and its applications in tasks such as zero-shot recognition and image segmentation. Explore the relationship between zero-shot learning and transfer learning across various computer vision domains.
* **Unit 12 - Ethics and Biases in Audio and Computer Vision** : understand the ethical considerations specific to computer vision. Explore why ethics matter, how biases can infiltrate AI models, and the types of biases prevalent in these domains. Learn how to do bias evaluation and mitigation strategies, emphasizing responsible development and deployment of AI technologies.

> * **Unit 13 - Outlook and Emerging Trends** : explore current trends and emerging architectures . Delve into innovative approaches like Retentive Network, Hiera, Hyena, I-JEPA, and RMT.

Reviewer 4: ```suggestion
* **Unit 13 - Outlook and Emerging Trends** : explore current trends and emerging architectures . Delve into innovative approaches like Retentive Network, Hiera, Hyena, I-JEPA, and Retention Vision Models.
```
RMT is just one model discussed in this chapter besides ViR, and both are about using retention for vision models.

 -  Writers: [William Bonvini](https://github.com/WilliamBonvini), [Alper Balbay](https://github.com/alperiox), [Madhav Kumar](https://github.com/miniMaddy), [Bhavesh Misra](https://github.com/Zekrom-7780)

 **Unit 11 - Zero Shot Computer Vision** 
 - Reviewers: [Mohammed Hamdy](https://github.com/mmhamdy)
 - Writers: [Mohammed Hamdy](https://github.com/mmhamdy) Writer and Reviewer: [Albert Kao](https://github.com/albertkao227)

 **Unit 12 - Ethics and Biases in Audio and Computer Vision** 
 - Reviewers: [Ratan Prasad](https://github.com/ratan), [Mohammed Hamdy](https://github.com/mmhamdy), [Charchit Sharma](https://github.com/charchit7), [Adhi Setiawan](https://github.com/adhiiisetiawan), [Ameed Taylor](https://github.com/atayloraerospace), [Bhavesh Misra](https://github.com/Zekrom-7780) 
 - Writers: [Snehil Sanyal](https://github.com/snehilsanyal), [Bhavesh Misra](https://github.com/Zekrom-7780)
  
 **Unit 13 - Outlook and Emerging Trends** 
 - Reviewers: [Ratan Prasad](https://github.com/ratan), [Ameed Taylor](https://github.com/atayloraerospace), [Mohammed Hamdy](https://github.com/mmhamdy) 

>  - Writers: [Farros Alferro](https://github.com/farrosalferro), [Mohammed Hamdy](https://github.com/mmhamdy)

Reviewer 4: ```suggestion
 - Writers: [Farros Alferro](https://github.com/farrosalferro), [Mohammed Hamdy](https://github.com/mmhamdy), [Louis Ulmer](https://github.com/lulmer), [Dario Wisznewer](https://github.com/dariowsz), [gonzachiar](https://github.com/gonzachiar)
```
Probably they've missed the contribution sheet, but @lulmer, @dariowsz, and @gonzachiar did contribute to writing this unit.

Author: @mmhamdy I will commit the suggestion, but can you fill in their contributions in the sheet?

Reviewer 4: Sure, no problem.

It is also a good way to motivate yourself to follow the course. Joining our community is an excellent way to stay engaged.  Who knows what is the next thing we will build together? 

As AI continues to advance, so does the quality of our discussions and the diversity of perspectives within our community. Upon becoming a member, you'll have an opportunity to connect with fellow course participants, exchange ideas, and collaborate with others. Moreover, the  contributors to this course are active on Discord and might help you when needed. Join us now!

## Computer Vision Channels
There are many channels focused on various topics on our Discord server. You will find people discussing papers, organizing events, sharing their projects and ideas, brainstorming, and so much more.

As a computer vision course learner, you may find the following set of channels particularly relevant:

* `#computer-vision`: a catch-all channel for everything related to computer vision.
* `#cv-study-group`: a place to exchange ideas, ask questions about specific posts, and start discussions.
* `#3d`: a channel to discuss aspects of computer vision specific to 3D computer vision

> If you are interested in generative AI, we also invite to join all channels related to the Diffusion Models: #core-announcements, #discussions, #dev-discussions, and #diff-i-made-this.

Reviewer 5: invite you?

## What you will  learn

The course is composed of theory, practical tutorials, and engaging challenges.

* **Theory Part** : This section covers the theoretical principles of computer vision, explained in detail with practical examples.
* **Hands-on Tutorials** : You will learn how to train and apply key computer vision models using Google Colab notebooks.

To illustrate what these computer vision models can achieve, here's a simple demo of a cat vs. dog classifier created with [Gradio](https://www.gradio.app/).

<iframe
	src="https://huggingface.co/spaces/ak0601/cat_dog_classifier"
	frameborder="0"
	width="850"
	height="450">
</iframe>

## Course Structure

> The course is organized into multiple units, covering the fundamentals and delving into in-depth exploration of state-of-the-art models.

Reviewer 5: delving into **an** in-depth exploration

typo?
